BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/SkontoViewModel.swift

```diff
@@ -98,26 +98,30 @@ class SkontoViewModel {
         notifyStateChangeHandlers()
     }
 
-    func setSkontoPrice(price: String) {
-        guard let price = convertPriceStringToPrice(price: price), price.value <= amountToPay.value else {
+    func setSkontoAmountToPayPrice(_ price: String) {
+        guard let price = convertPriceStringToPrice(price: price),
+              price.value <= amountToPay.value else {
             notifyStateChangeHandlers()
             return
         }
         skontoAmountToPay = price
+        updateDocumentPagesModelData()
         recalculateSkontoPercentage()
         notifyStateChangeHandlers()
     }
 
-    func setDefaultPrice(price: String) {
+    func setAmountToPayPrice(_ price: String) {
```

Comment:
[Suggestion] Probably we can specify it as `setFullAmount` and for Skonto `setFinalAmount` - it will be the same namings as we have in UI labels for this textFields

Reply:
To be honest for me "full amount" and "final amount" are not very clear which are in the UI without context, as well those might be renamed at some point and we will lose the sense here. So I would prefer to keep `AmountToPay` and `SkontoAmountToPay`.